lib/sass.js

```diff
@@ -36,14 +36,14 @@ export async function buildCSS(inputs, { tailwindConfig } = {}) {
 
   await Promise.all(
     inputs.map(async input => {
-      const output = `${outDir}/${basename(input, extname(input))}.css`;
-      const sourcemapPath = output + '.map';
+      const inputFile = `${basename(input, extname(input))}.css`;
+      const output = `${outDir}/${inputFile}`;
+      const sourcemapPath = `${output}.map`;
 
-      const sassResult = sass.renderSync({
-        file: input,
-        includePaths: [dirname(input), 'node_modules'],
-        outputStyle: minify ? 'compressed' : 'expanded',
-        sourceMap: sourcemapPath,
+      const sassResult = sass.compile(input, {
+        loadPaths: [dirname(input), 'node_modules'],
+        style: minify ? 'compressed' : 'expanded',
+        sourceMap: true,
```

Comment:
I have been testing this in `client`, and diffing generated results. This generates equivalent results, with some empty lines added/removed as the only difference.

Incidentally, I have found out we are actually not minifying our bundles for production. I suspect it's because we minify with sass, but then postcss unminifies it, so we may need to move this to step to a postcss plugin instead.

Reply:
> I suspect it's because we minify with sass, but then postcss unminifies it,

Does it make any difference at all to the production output whether SASS minification is enabled?

Most of the CSS is generated by Tailwind nowadays, which is a PostCSS plugin. Since that is added after SASS processing, I'd expect it to be unaffected.

I can imagine that PostCSS might "undo" some aspects of minification for the content that _is_ produced by SASS as it parses the input into an AST and then re-serializes it later. Depending on what SASS minification does, it might not undo all aspects however.

Reply:
Oh, that's actually a good point. I think I somehow expected to see a "oneliner" CSS output, and since the first few lines did not look minified, I thought none of it was.

Let me double check this. Perhaps you are right, and part of the output is indeed minified.

Reply:
Yep, you are right. The result of calling `sass.compile`/`sass.renderSync` does minify correctly.

Only tailwind stuff that's added afterwards is not minified.

I'll create a separated issue so that we can discuss how to proceed.